src/connectors/configManager/impl.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const _ = require('lodash');
+const assert = require('assert');
+const Connector = require('../Connector');
+
+const { host } = require('../../config').configManager;
+
+const metrics = require('../metrics');
+
+module.exports = new class extends Connector {
+    constructor () {
+        super(module);
+        this.currentState = metrics.createConnectorMetric(this.getName(), 'getCurrentState');
+    }
+
+    async getCurrentState () {
+        const uri = this.buildUri(host, 'api', 'config-manager', 'v1');
+        uri.segment('states');
+        uri.segment('current');
+
+        const options = {
+            uri: uri.toString(),
+            method: 'GET',
+            json: true,
+            headers: this.getForwardedHeaders()
+        };
+
+        const result = await this.doHttp (options, false, this.fetchRunHosts);
```

Comment:
Should this be passing the result to fetchRunHosts? (I haven't looked into how this works past the PR)

Reply:
nope!  Just fixed it!  :)